examples/signeduploads/sampleapp/views.py

```diff
@@ -0,0 +1,23 @@
+from django.shortcuts import render
+from django.http import HttpResponse
+import json
+import time
+from .forms import ImgForm
+from pyuploadcare import generate_secure_signature
+from django.conf import settings
+from django.views.decorators.csrf import csrf_exempt
+
+
+def index(request):
+    imgForm = ImgForm()
+    return render(request, 'sampleapp/index.html', {'form': imgForm})
+
+
+@csrf_exempt
+def release_signature(request):
```

Comment:
planning to add here auth-based views, and assign signatures to users based on who is signed-in, therefore "release_signature".